portfolio/src/main/webapp/blog.html

```diff
@@ -0,0 +1,105 @@
+<html>
+
+<head>
+    <meta charset="UTF-8" />
+    <title>Gaurnett Flowers Portfolio</title>
+    <link rel="stylesheet" href="./css/style.css" />
+
+    <!-- Font Awesome Icons -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+    <!--Import Google Icon Font-->
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+        integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous" />
+
+    <!--Let browser know website is optimized for mobile-->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <!-- jQuery Google Host -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+</head>
+
+<body>
+    <!-- Navigation -->
+    <header id="blog-banner">
+        <nav class="navbar navbar-expand-lg fixed-top py-3">
+            <div class="container">
+                <button type="button" data-toggle="collapse" data-target="#nav-items" aria-controls="nav-items"
+                    aria-expanded="false" aria-label="Toggle navigation" class="navbar-toggler navbar-toggler-right">
+                    <i class="fa fa-bars"></i>
+                </button>
+
+                <div id="nav-items" class="collapse navbar-collapse">
+                    <ul class="navbar-nav ml-auto mx-auto">
+                        <li class="nav-item">
+                            <a href="index.html" class="nav-text-scrolled nav-link text-uppercase font-weight-bold">Gaurnett Flowers</a>
+                        </li>
+                        <li class="nav-item">
+                            <a href="blog.html" class="nav-text-scrolled nav-link text-uppercase font-weight-bold">Blog</a>
+                        </li>
+                        <li class="nav-item">
+                            <a href="gallery.html" class="nav-text-scrolled nav-link text-uppercase font-weight-bold">Gallery</a>
+                        </li>
+                        <li class="nav-item">
+                            <a target="_blank" href="https://github.com/gaurnett" class="nav-text-scrolled nav-link"><i class="fa fa-github" style="font-size:24px"></i></a>
+                        </li>
+                        <li class="nav-item">
+                            <a target="_blank" href="https://www.linkedin.com/in/gaurnett-flowers-9a5642154" class="nav-text-scrolled nav-link"><i class="fa fa-linkedin" style="font-size:24px"></i></a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </nav>
+    </header>
+
+    <!-- Projects Section -->
+    <section>
+        <div class="container">
+            <div class="row">
+                <div class="col-1"></div>
```

Comment:
It looks like you may be using these `col-1` divs just to offset the `col-10` container. If this is the case, there's actually an offset class that bootstrap gives you for free!

Check out: https://getbootstrap.com/docs/4.0/layout/grid/#offsetting-columns